Source/WebCore/platform/graphics/win/MediaPlayerPrivateMediaFoundation.cpp

```diff
@@ -87,7 +87,7 @@ class MediaPlayerPrivateMediaFoundation::AsyncCallback : public IMFAsyncCallback
         return refCount;
     }
 
-    HRESULT STDMETHODCALLTYPE GetParameters(__RPC__out DWORD *pdwFlags, __RPC__out DWORD *pdwQueue) override
+    HRESULT STDMETHODCALLTYPE GetParameters(__RPC__out DWORD *, __RPC__out DWORD *) override
```

Comment:
Would it be worth removing the space before the `*` in these cases?

Reply:
Will fix. Thank you.